### DISCUSSION TITLE
Is this for local severs only or can I use it for like 2b2t or something like that?

### DISCUSSION THREAD

**Discussion by Logano143** [mircokroon/minecraft-world-downloader#369] [Q&A] [answered]
The reason I ask is that in the instructions and in the app it keeps talking about Local host and stuff does that mean it's just local things?
I have gotten to this point but cant really find anything about the local host things.
![image](https://user-images.githubusercontent.com/106934377/172074227-317e5e14-cce7-43b9-ba7f-4ec483cb4fe5.png)

**Comment by mircokroon** [comment, marked as answer]
You connect to the server address `localhost` in Minecraft, which will then connect you to the server given in the menu. You can use it for servers like 2b2t.

**Reply by Logano143** [reply]
By sever address localhost do you mean like the sever address then after it put localhost?
![image](https://user-images.githubusercontent.com/106934377/172156656-c24a88af-d12b-43ed-9c34-df949530fa85.png)
 It doesn't work though, it says unknown host when trying to join.

**Reply by mircokroon** [reply]
No, _just_ `localhost`. You don't have to repeat the server address.

**Reply by Logano143** [reply]
Ok, I see now thanks!
